pages/intro.js

```diff
@@ -0,0 +1,197 @@
+import Card, { CardHeader, CardTitle, CardContent, CardButtonContainer } from '../components/card'
+import styled from 'styled-components'
+import { useState, useEffect } from 'react'
+import { COLOR, EDIT } from '../constants'
+import Button from '../components/button'
+
+const props = {
+    nwHacks: {
+        subtitle: `Last modified [TIME STAMP FIREBASE] by [USER FROM FIREBASE]`,
+        contentHeader: `This is nwHacks 2019`,
+        content: `Come make things and break things, and then make them cooler. You'll never be short on inspiration when you're surrounded by 650 of the brightest minds in the Pacific Northwest. All you need to bring is an open mind and an insatiable desire to learn; we'll take care of the rest. After all, we're western Canada's largest hackathon - we make the west coast the best coast.`,
+        otherSubtitle: `Last modified [TIME STAMP FIREBASE] by [USER FROM FIREBASE]`,
+        otherContentHeader: `Why nwHacks?`,
+        otherContent: `Vancouver is breathtaking and so are you`
+    }
+}
+
+const HeaderText = styled.textarea`
+    border: none;
+    width: 847px;
```

Comment:
using hardcoded width isn't very responsive. I'd suggest adding padding to the parent and setting these `textarea`s to `display: block` (I think that should force them to 100% width)

https://stackoverflow.com/questions/17468733/difference-between-width-auto-and-width-100-percent#:~:text=up%20vote%202-,The%20initial%20width%20of%20a%20block%20level%20element%20like%20div,horizontal%20margin%2C%20padding%20and%20border.